server/src/main/java/org/elasticsearch/search/SearchService.java

```diff
@@ -216,6 +225,14 @@ private void setDefaultSearchTimeout(TimeValue defaultSearchTimeout) {
         this.defaultSearchTimeout = defaultSearchTimeout;
     }
 
+    private void setDefaultAllowPartialSearchResults(Boolean defaultAllowPartialSearchResults) {
```

Comment:
We should be able to make this a `boolean`? since there is always a value returned even if the user has not set it themselves?